### PR TITLE
Added recipe for shears to iron in the crucible

### PR DESCRIPTION
**Pull request by DeGariless**

### CODE REVIEW

File: src/main/java/betterwithmods/BWCrafting.java [hunk 1]
```diff
@@ -342,6 +342,7 @@ private static void addCrucibleRecipes() {
         addStokedCrucibleRecipe(new ItemStack(Items.IRON_INGOT, 8, 0), new ItemStack[]{new ItemStack(Items.IRON_CHESTPLATE, 1, OreDictionary.WILDCARD_VALUE)});
         addStokedCrucibleRecipe(new ItemStack(Items.IRON_INGOT, 7, 0), new ItemStack[]{new ItemStack(Items.IRON_LEGGINGS, 1, OreDictionary.WILDCARD_VALUE)});
         addStokedCrucibleRecipe(new ItemStack(Items.IRON_INGOT, 2, 0), new ItemStack[]{new ItemStack(Items.IRON_DOOR)});
+        addStokedCrucibleRecipe(new ItemStack(Items.IRON_INGOT, 3), new ItemStack[]{new ItemStack(Items.SHEARS)});
```
Comment (primetoxinz) [inline src/main/java/betterwithmods/BWCrafting.java]: should be 2 iron, not 3

Comment (DeGariless) [inline src/main/java/betterwithmods/BWCrafting.java]: Roops XD

Comment (BeetoGuy) [inline src/main/java/betterwithmods/BWCrafting.java]: Shears also need the wildcard value.

File: src/main/java/betterwithmods/blocks/BlockKiln.java [hunk 2]
```diff
@@ -234,8 +234,23 @@ private boolean checkKilnIntegrity(IBlockAccess world, BlockPos pos)
...
+            case 3:  toCheck = pos.up(2);
+                     break;
+            case 4:  toCheck = pos.up().east();
+                     break;
+            case 5:  toCheck = pos.up().north();
+                     break;
+            case 6:  toCheck = pos.up().south();
+                     break;
+			}
+			Block block = world.getBlockState(toCheck).getBlock();
```
Comment (BeetoGuy) [inline src/main/java/betterwithmods/blocks/BlockKiln.java]: I think this can easily be solved by pos.up().offset(i). Seems a bit more tidy that way.

Comment (DeGariless) [inline src/main/java/betterwithmods/blocks/BlockKiln.java]: Alright. Still learning my way around forge (and github). Thanks for the tips.

**Comment by DeGariless** [comment]
I think this is what you were looking for.

**Pull request merged**